prac_05/state_names.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+
```

Comment:
Hey, just to let you know code to name isn't in pep8 formatting, according to pep8 dictionaries should go {"key": "value"} with no space for the key and one space for the value